torch/csrc/jit/tensorexpr/ir.cpp

```diff
@@ -144,6 +145,14 @@ Store* Store::make(
       ExprHandle(1).node());
 }
 
+Allocate* Allocate::make(
+    const VarHandle& buffer_var,
+    Dtype dtype,
+    const std::vector<ExprHandle>& dims) {
```

Comment:
I think we should remove API for creating `Allocate` statements with `Var` - the API taking `Buf` should handle everything we need.